docs/get-started.md

```diff
@@ -51,7 +51,7 @@ Also update the Babel configuration in the `package.json` file:
 
 4. Install `react-rails`:
 ```bash
-$ bundle add 'react-rails' --strict
+$ bundle add 'react-rails' --strict --version '3.1.1'
```

Comment:
This way, we should update the docs for every single release. Isn't it?

Reply:
hmm how about like `bundle add 'react-rails' --strict --version '>=3.1.1'`

Reply:
It is important to install "exact" version for both the Ruby gem and node package. This is emphasized in the documentation as well.